modules/javafx.web/src/main/java/com/sun/javafx/webkit/drt/UIClientImpl.java

```diff
@@ -182,7 +183,30 @@ public boolean runBeforeUnloadConfirmPanel(String message) {
      */
     @Override
     public String[] chooseFile(String initialFileName, boolean multiple, String mimeFilters) {
-        throw new UnsupportedOperationException("Not supported yet");
+        if (!DumpRenderTree.drt.complete()) {
```

Comment:
The entire function should be skipped if `DumpRenderTree.drt.complete()` is true, not just the print.